client/src/components/DataFiles/DataFilesModals/DataFilesAddProjectModal.jsx

```diff
@@ -102,7 +102,7 @@ const DataFilesAddProjectModal = () => {
         otherwise: (schema) => schema.notRequired(),
       }),
     keywords: Yup.string().matches(
-      /^\w+(\s*,\s*\w+)*$/,
+      /^\s*[\w-]+(\s*,\s*[\w-]+)*\s*$/,
```

Comment:
If we accept leading and trailing whitespaces, do we later trim the keywords field before saving to the db?

Reply:
I put it on the add and edit modals before they are dispatched.